pkg/distributor/query_ingest_storage_test.go

```diff
@@ -489,66 +489,71 @@ func TestDistributor_QueryStream_ShouldSupportIngestStorage(t *testing.T) {
 
 	for testName, testData := range tests {
 		testData := testData
+		for _, disableStreamingResponse := range []bool{false, true} {
```

Comment:
I might have led you astray in our conversation sorry - we don't need this backwards compatibility test (with streaming disabled) for this ingest storage test case, as anything that supports ingest storage will support streaming.

Reply:
I have updated the test. Thank you.